### ISSUE TITLE
Home when 

### ISSUE THREAD

**Issue by mojocorp** [mcheah/Marlin4MPMD#24]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by mojocorp** [comment]
Thank you so much for this amazing work ! It's working perfectly so far, way better than the original firmware. You just saved me a lot of pain. Thank you again!

**Comment by mcheah** [comment]
Yup, this is a known issue.  Last time I tried it broke things.  If you're running octoprint you can add a on-cancel g-code script that will do this for now

**Comment by Krishpannze** [comment]
I am using cura what should i do
